org.freeciv.gtk322.yml

```diff
@@ -0,0 +1,32 @@
+# Based on manifest on Freeciv tree, with minimal changes.
+# In the future we may automate the process of using that
+# "nightly builds" manifest from the release, or otherwise
+# arrange things so that we don't need to maintain these
+# separately - so try not to deviate too much.
+
+app-id: org.freeciv.gtk322
```

Comment:
Given that there is `org.freeciv.Freeciv` this name feels out of place if it to provide a 3.0x client along the existing 2.6.7

(did I get that right? that client are incompatible? https://github.com/flathub/org.freeciv.Freeciv/issues/8 )

Reply:
Yes, freeciv-2.6 and freeciv-3.0 are incompatible. Likely the reason people behind org.freeciv.Freeciv flatpak have not updated is that they want to keep using 2.6 servers.
The idea of the naming here is that the last part should tell what freeciv.org client or utility it really is about. Separate flatpaks for Qt-client and sdl2-client are expected to come soon (and gtk4-client one along with freeciv-3.1). It's yet clear whether some utilities will also come as separate flatpaks, or will they be merged to client flatpaks.

Reply:
So because someone submitted a flatpak named org.freeciv.Freeciv before it was your policy that upstream should be in control (as seen in above "I own the domain used in the application ID" term for new submissions), we can't now have properly namespaced ones? This is getting a bit frustrating.

Reply:
just trying to avoid duplication here. I don't know the origin of the other package, but it doesn't seem to have a maintainer per-see.

Reply:
The way I see it is that @barthalion can also provide you maintainer access to the other repo. It can be deprecated and archived if you desire, once this one approved.

Reply:
> The way I see it is that @barthalion can also provide you maintainer access to the other repo. It can be deprecated and archived if you desire, once this one approved.

Yeah, that's one way to go, though I'm not sure if it's letting some users of that other repo down if they have been promised freeciv-2.6 compatible flatpak (a lot of online servers are still 2.6 ones).
In any case, grant any such access to @freeciv-github account, not to my personal account.